lib/rules/at-rule-empty-line-before/index.js

```diff
@@ -133,7 +135,7 @@ function rule(expectation, options, context) {
 			}
 
 			// Fix
-			if (context.fix) {
+			if (context.fix && context.newline) {
```

Comment:
[note] `context.newline` can be `undefined`, so type-check raises an error without this addition.